### PR TITLE
Filter blog collection by language

### PR DESCRIPTION
**Pull request by devnook**
On a localized blog home page (/pl/blog) we want to show blog posts in particular language (e.g. pl).

### CODE REVIEW

File: src/site/content/en/lang.11tydata.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (robdodson) [inline src/site/content/en/lang.11tydata.js:1]: Does this file work? Eleventy's docs say:

image

https://www.11ty.dev/docs/data-template-dir/

Comment (robdodson) [reply src/site/content/en/lang.11tydata.js:1]: I opened an issue about it a while back https://github.com/11ty/eleventy/issues/492

**Comment by robdodson** [comment]
Percy failure can be ignored. It's just the top nav underline being flaky
image

File: src/site/content/en/lang.11tydata.js (hunk 1 above)
Comment (MichaelSolati) [reply src/site/content/en/lang.11tydata.js:1]: I don't think it'll trickle to other files, but seeing how it's being required in the blog it should be fine.

Comment (devnook) [reply src/site/content/en/lang.11tydata.js:1]: Actually it does not need to be an 11tydata file, normal .js would do too.

**Comment by robdodson** [comment]
@MichaelSolati @devnook Circling back to this. I was wondering... why have a `lang.js` file when we could put that info into the `en.11tydata.js` file and have other files require `en.11tydata.js` if they need to access it?

**Pull request merged**